src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -149,11 +127,11 @@ public function testValidateIfChildWithValidConstraint()
         $this->assertNoViolation();
     }
 
-    public function testDontValidateIfParentWithoutCascadeValidation()
+    public function testDontValidateIfParentWithoutValidConstraint()
```

Comment:
Seems so, but I am actually a bit confused about the difference of `testValidateConstraintsEvenIfNoValidConstraint` and `testDontValidateIfParentWithoutValidConstraint` as it looks like in the first tests constraints in the child are validated even when there is no Valid constraint.